app/graphql/mutations/user/login.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# Mutation to authenticate and login a user, generating a token
+module Mutations
+  module User
+    class Login < Mutations::BaseMutation
+      description "Login a user"
+
+      # RelayClassicMutation only accepts single input
+      argument :input, Inputs::User::Login, required: true
+
+      field :token, String, null: true
+      field :user, Types::Models::UserType, null: true
+
+      def resolve(input: nil)
+        user = User.authenticate(**input)
+
+        return {} unless user
+
+        raise(GraphQL::ExecutionError.new("User not verified")) unless
+          user.verified
+
+        now = Time.zone.now
+        args = {
+          last_login: now,
+          updated_at: now
```

Comment:
updated_at should automagically be updated for you.

Reply:
ok. Do i understand right, that if I just do `User.update({ last_login: now })` this is all I need to do, and will update `updated_at`? If so I will delete that second arg.